collections/business/general-notifications/event-general-notifications/account-balance-changed/account-balance-low/push/push_channel-settings.json

```diff
@@ -12,7 +12,6 @@
       "settings": {
```

Comment:
See this comment: https://github.com/Backbase/engagements-data/pull/27#issuecomment-1591125199